Block/NodeType/Product.php

```diff
@@ -225,4 +240,24 @@ public function getLabel()
     {
         return __("Product");
     }
+
+    /**
+     * @param int $nodeId
+     * @return false|string
+     * @throws \InvalidArgumentException
+     */
+    public function getProductTitle($nodeId)
+    {
+        return $this->getProductData($this->productTitles, $nodeId);
+    }
+
+    /**
+     * @param float $amount
+     * @return string
+     * @throws \InvalidArgumentException
```

Comment:
This tag is unnecessary, because you don't throw exception in this method directly.